decidim-admin/app/views/decidim/admin/moderations/reports/show.html.erb

```diff
@@ -1,61 +1,60 @@
+      <div class="current">
+        <%= reported_content_for moderation.reportable, locale: @report.locale %>
+      </div>
+      <% if translatable_resource?(moderation.reportable) %>
+        <div class="original">
+          <%= reported_content_for moderation.reportable, locale: moderation.reportable.content_original_language %>
         </div>
-        <% if translatable_resource?(moderation.reportable) %>
-          <div class="original">
-            <%= reported_content_for moderation.reportable, locale: moderation.reportable.content_original_language %>
-          </div>
-        <% end %>
-      </dd>
+      <% end %>
+    </dd>
 
-      <dt><%= t(".report_reason") %></dt>
-      <dd><%= t("decidim.admin.moderations.report.reasons.#{@report.reason}") %></dd>
+    <dt><%= t(".report_reason") %></dt>
+    <dd><%= t("decidim.admin.moderations.report.reasons.#{@report.reason}") %></dd>
 
-      <dt><%= t(".report_details") %></dt>
-      <dd><%= @report.details %>
-      </div>
-    </dl>
-    <div class="form__wrapper-block flex-col-reverse md:flex-row justify-between">
-      <% if !moderation.reportable.hidden? && allowed_to?(:unreport, permission_resource) %>
-        <%= link_to t("actions.unreport", scope: "decidim.moderations"), unreport_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary" %>
-      <% end %>
-      <% if !moderation.reportable.hidden? && allowed_to?(:hide, permission_resource) %>
-        <%= link_to t("actions.hide", scope: "decidim.moderations"), hide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
-      <% end %>
-      <% if moderation.reportable.hidden? && allowed_to?(:unhide, permission_resource) %>
-        <%= link_to t("actions.unhide", scope: "decidim.moderations"), unhide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
-      <% end %>
-    </div>
+    <dt><%= t(".report_details") %></dt>
+    <dd><%= @report.details %></dd>
+  </dl>
+  <div class="form__wrapper-block flex-col-reverse md:flex-row justify-between">
+    <% if !moderation.reportable.hidden? && allowed_to?(:unreport, permission_resource) %>
+      <%= link_to t("actions.unreport", scope: "decidim.moderations"), unreport_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary" %>
+    <% end %>
+    <% if !moderation.reportable.hidden? && allowed_to?(:hide, permission_resource) %>
+      <%= link_to t("actions.hide", scope: "decidim.moderations"), hide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
+    <% end %>
+    <% if moderation.reportable.hidden? && allowed_to?(:unhide, permission_resource) %>
+      <%= link_to t("actions.unhide", scope: "decidim.moderations"), unhide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
+    <% end %>
```

Comment:
Same here, I think that if you're in this page you want to have really clear that you can hide/unhide the thing that's being reported